fillers/withdrawals/withdrawals.py

```diff
@@ -77,7 +79,7 @@ def test_withdrawals_use_value_in_tx(_):
         index=0,
         validator=0,
         address=TestAddress,
-        amount=tx.gas_price * tx.gas_limit + 1,
+        amount=(tx.gas_price * tx.gas_limit + ONE_GWEI),
```

Comment:
There is a problem when specifying `(tx.gas_price * tx.gas_limit)` here, because their unit is wei, but `withdrawal.amount` expects gwei, and this multiplication actually translates to `(tx.gas_price * tx.gas_limit * 1e9)` in post's balance, so the test is failing.

One trick could be to set the gas_price to `ONE_GWEI` and remove `tx.gas_price` from this formula.

Another thing to consider is that, when we use `ONE_GWEI` in the withdrawal amount, it translates to `ONE_GWEI * ONE_GWEI` in the post balance, so we need to use a `1` here and check for `ONE_GWEI` in post.

Reply:
Apologies for this, a simple error on my part, after fixing the errors I realised I misunderstood the conversions. I've updated the PR with the commits below